jodconverter-core/src/main/java/org/artofsolving/jodconverter/document/DefaultDocumentFormatRegistry.java

```diff
@@ -125,6 +125,11 @@ public DefaultDocumentFormatRegistry() {
         tsv.setStoreProperties(DocumentFamily.SPREADSHEET, tsvLoadAndStoreProperties);
         addFormat(tsv);
 
+        DocumentFormat ots = new DocumentFormat(" OpenDocument Spreadsheet Template", "ots", "application/vnd.oasis.opendocument.spreadsheet-template");
```

Comment:
Thank you @montgolfiere for the fyi 🙏 

I believe there might be an additional unwanted whitespace here?
```suggestion
        DocumentFormat ots = new DocumentFormat("OpenDocument Spreadsheet Template", "ots", "application/vnd.oasis.opendocument.spreadsheet-template");
```

Reply:
Good spot ! Resolved by @srimantahyland